apis/cosmos-reducers.js

```diff
@@ -619,7 +619,7 @@ export function transactionReducer(transaction) {
           transaction
         ),
         timestamp: transaction.timestamp,
-        memo: transaction.tx.value.memo,
+        memo: transaction.tx.body.memo,
```

Comment:
Now it will probably fail? :eyes:

Reply:
the reducers are for stargate... the cosmoshub-3 reducers are cosmos-reducers-v39